budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/budgets/components/targetactualchart/BudgetsWeeklyAggregationModel.java

```diff
@@ -31,9 +31,11 @@ public BudgetsWeeklyAggregationModel(IModel<BudgetTagFilter> filterModel) {
     @Override
     protected TargetAndActual load() {
         if (budgetId != 0) {
-            return service.getWeekStatsForBudget(budgetId, 12);
+            return service.getWeekStatsForBudgetWithTax(budgetId, 12);
+            //return service.getWeekStatsForBudget(budgetId, 12);
         } else if (filterModel != null && filterModel.getObject() != null) {
-            return service.getWeekStatsForBudgets(filterModel.getObject(), 12);
+            //return service.getWeekStatsForBudgets(filterModel.getObject(), 12);
```

Comment:
Also this one and the other ones in `BudgetsMonthlyAggregatedRecordsModel` and `BudgetsWeeklyAggregatedRecordsModel`